### DISCUSSION TITLE
Version 4.3.0 (for Minecraft 1.20.2-1.20.4)

### DISCUSSION THREAD

**Discussion by lina-x64** [DAMcraft/MeteorServerSeeker#35] [Announcements]
- Historic players hud
- Fixed crash if you had the default discord pfp
- Added the ability to save settings

<hr /><em>This discussion was created from the release <a href='https://github.com/DAMcraft/MeteorServerSeeker/releases/tag/4.3.0'>Version 4.3.0 (for Minecraft 1.20.2-1.20.4)</a>.</em>

**Comment by ByfronTechnologies** [comment]
this shi fire!!!! no cap
